packages/material-ui/src/TableCell/TableCell.test.js

```diff
@@ -30,52 +35,65 @@ describe('<TableCell />', () => {
   describeConformance(<TableCell />, () => ({
     classes,
     inheritComponent: 'td',
-    mount: mountInTable,
+    mount: (node) => {
```

Comment:
I think this is now not cleaned up, right? I think you need to add `after: () => mount.cleanUp()`

Reply:
The cleanUp is before the describeConformance

Reply:
Right! Can we move this into `describeConformance(...m () => ({ after }))` or are we still using `mount` somewhere else?

Reply:
yes we can move